cdap-proto/src/main/java/io/cdap/cdap/proto/security/SpecialPermission.java

```diff
@@ -0,0 +1,49 @@
+/*
```

Comment:
I think SpecialPermission may be unnecessary since we have full control over both the caller and callee during the permission check process, so if the authorization enforcer encounters a permission which doesn't map to any permission, we can log and return an error for those.

Reply:
Removed